js/widgets/widget.theme.js

```diff
@@ -29,17 +29,30 @@
 
 $.mobile.widget.theme = {
 	_create: function() {
+		var that = this;
 		this._super();
 		if ( !this.options.enhanced && this.options.theme ) {
```

Comment:
What about ```this.options.somethingTheme```? You should go into this loop even if ```this.options.theme``` is not set, because ```this.options.somethingTheme``` may be set.

This may also reduce startup performance. In the popup widget I make sure to add all the classes after I've detached ```this.element``` from the DOM, but before I re-attach it as part of the popup. If I were to use this, I'd have to call super() at just the right time.